MIGRATION.md

```diff
@@ -3,6 +3,13 @@
 The main goal of version 7 is to reduce bundle size. This version is breaking because we removed deprecated APIs, upgraded our build tooling, and restructured npm package contents.
 Below we will outline all the breaking changes you should consider when upgrading.
 
+TL;DR if you copy & pasted the basic examples from our docs:
+- Our CDN bundles are now ES6 - you will need to [reconfigure your script tags](#renaming-of-cdn-bundles) if you want to keep supporting ES5 and IE11 on the new SDK version.
+- Distributed CommonJS files will be ES6. Use a transpiler if you need to support old node versions.
+- We bumped the TypeScript version we generate our types with to 3.8.3. Please check if your TypeScript projects using TypeScript version 3.7 or lower still compile. Otherwise, upgrade your TypeScript version.
+- `whitelistUrls` and `blacklistUrls` have been renamed to `allowUrls` and `denyUrls` in the `Sentry.init()` options.
+- The `UserAgent` integration is now called `HttpContext`.
```

Comment:
I wonder if we need `UserAgent` in this list. Is it being referenced a lot in docs? But given the size of the list I think it's fine if we leave it in.

Reply:
I had exactly the same thought. I think it's in the docs once, which I agree is not very prominent. My gut said to include it - couldn't really make a decision based on rules or data 🤷‍♂️. Let's just leave it and remove it if we get feedback that it's too confusing.

Reply:
Sounds good to me!